src/main/resources/jsonSchemas/antoraComponentSchema.json

```diff
@@ -2,8 +2,7 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Antora component descriptor antora.yml",
   "required": [
-    "name",
-    "version"
```

Comment:
What we can't express with two separate schemas: A version must be set in at least one of the two possible locations, playbook or component description, for each component. Otherwise, antora exits with a FATAL error.

Should that be validated using logic in the Java code?

Reply:
Never mind, the two files would rarely live in the same project, and IDEA might not have access to the Git projects as specified.

Reply:
Sometimes the playbook has attributes defined that the component doesn't know about. In such a case it might be helpful to setup a project that contains both repositories. This way the IDE will know about the attribute and use them in the preview / in autocomplete. See https://intellij-asciidoc-plugin.ahus1.de/docs/users-guide/features/advanced/multimodule.html for details.